skills/declarative/People/People/People.dialog

```diff
@@ -214,8 +206,8 @@
           "activityProcessed": true,
           "dialog": "GetDirectReportsDialog",
           "options": {
-            "nameEntity": "=$options.contactEntity",
```

Comment:
Any particular reasons why we are ditching the options variable? Is it just for understandability?

Reply:
Yeah, since there are only two properties it doesn't help that much